notebooks/run_model_in_notebook.ipynb

```diff
@@ -46,7 +46,7 @@
     "# path to config file\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    import PyStemmusScope
</pre><p>Please add these two lines to<code> PyStemmusScope/<strong>__init__.py.</strong></code></p><p><code>from . import iostreamer</code></p><p><code>from . import forcing_io</code></p><p> </p><p>and use them in all notebooks as :</p><p> </p><p><code>from PyStemmusScope import iostreamer</code></p><p><code>from PyStemmusScope import forcing_io </code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/STEMMUS_SCOPE_Processing/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>